nativelib/src/main/resources/gc/commix/Marker.c

```diff
@@ -0,0 +1,367 @@
+#include <stdio.h>
+#include <setjmp.h>
+#include "Marker.h"
+#include "Object.h"
+#include "Log.h"
+#include "State.h"
+#include "headers/ObjectHeader.h"
+#include "datastructures/GreyPacket.h"
+#include "GCThread.h"
+#include <sched.h>
+
```

Comment:
Needs a comment somewhere around here to explain the gray packets and how they are managed.